Userland/Libraries/LibWeb/DOM/ShadowRoot.idl

```diff
@@ -1,11 +1,15 @@
 #import <DOM/DocumentFragment.idl>
+#import <DOM/InnerHTML.idl>
 
+// https://dom.spec.whatwg.org/#shadowroot
 interface ShadowRoot : DocumentFragment {
-
-    readonly attribute DOMString mode;
+    // FIXME: mode should return a ShadowRootMode
+    //readonly attribute DOMString mode;
```

Comment:
`// readonly...`

Reply:
Oops, that shouldn't be commented at all! Fixed. :sweat_smile: